doc/whats_new/v1.4.rst

```diff
@@ -174,6 +174,11 @@ Changelog
   metric parameter `p` is in the range `0 < p < 1`, regardless of the `dtype` of `X`.
   :pr:`26760` by :user:`Shreesha Kumar Bhat <Shreesha3112>`.
 
+- |Efficiency| The performance of :meth:`neighbors.RadiusNeighborsClassifier.predict`
+  and of :meth:`neighbors.RadiusNeighborsClassifier.predict_proba` has been improved
+  when `radius` is large and `algorithm="brute"` with non-Euclidean metrics.
+  :pr:`26828` by :user:`Omar Salman <OmarManzoor>`.
```

Comment:
Eventhough we need to perform more comprehensive benchmark, your work has more impact than that: it also improves the performance of estimators which use those two methods under the hood, and we should also list them here.

Reply:
I am not sure RadiusNeighborsClassifier is used in other classifiers? I think we would need other classifiers to actually use RadiusNeighborsClassifier's predict_proba if we want to include them?

Reply:
You are right, while `RadiusNeighbors*.radius_neighbors` is used by the following estimator:

```
  - :class:`sklearn.cluster.MeanShift`
  - :class:`sklearn.cluster.OPTICS`
  - :class:`sklearn.neighbors.LocalOutlierFactor`
  - :class:`sklearn.manifold.Isomap`
```

`RadiusNeighborsClassifier.predict_proba` seems not to. You can ignore my comment.